mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/StartupCrashFlushScenario.kt

```diff
@@ -0,0 +1,26 @@
+package com.bugsnag.android.mazerunner.scenarios
+
+import android.content.Context
+import com.bugsnag.android.*
+
+/**
+ * Tests that both requests are sent in the case where 1 startup crash and 1 regular crash are
+ * persisted
+ */
+internal class StartupCrashFlushScenario(config: Configuration,
+                                         context: Context) : Scenario(config, context) {
+
+    override fun run() {
+        super.run()
+        disableAllDelivery()
+
+        if ("StartupCrash" == eventMetaData) {
+            config.launchCrashThresholdMs = Long.MAX_VALUE
```

Comment:
I think this needs some rethinking. I think you dont want to set `config.launchCrashThresholdMs` in here. Also both of these crashes will count as launch crashes on subsequent boots.